types/react-native-elements/index.d.ts

```diff
@@ -250,4 +400,1126 @@ export interface BadgeProperties {
+export class Badge extends React.Component<BadgeProps, any> {}
+
+export interface CardProps {
+    /**
+     * Flex direction (row or column)
+     *
+     * @default 'column'
+     */
+    flexDirection?: 'column' | 'row';
+
+    /**
+     * Outer container style
+     */
+    containerStyle?: ViewStyle;
+
+    /**
+     * Inner container style
+     */
+    wrapperStyle?: ViewStyle;
+
+    /**
+     * Card title
+     */
+    title?: string;
+
+    /**
+     * Additional title styling (if title provided)
+     */
+    titleStyle?: TextStyle;
+
+    /**
+     * Title rendered over the image
+     * (only works if image prop is present)
+     */
+    featuredTitle?: string;
+
+    /**
+     * Styling for featured title
+     */
+    featuredTitleStyle?: TextStyle;
+
+    /**
+     * Subtitle rendered over the image
+     * (only works if image prop is present)
+     */
+    featuredSubtitle?: string;
+
+    /**
+     * Styling for featured subtitle
+     */
+    featuredSubtitleStyle?: TextStyle;
+
+    /**
+     * Additional divider styling
+     * (if title provided)
+     */
+    dividerStyle?: ViewStyle;
+
+    /**
+     * Specify different font family
+     *
+     * @default System font bold (iOS), Sans Serif Bold (android)
+     */
+    fontFamily?: string;
+
+    /**
+     * Specify image styling if image is provided
+     */
+    imageStyle?: ImageStyle;
+
+    /**
+     * Specify styling for view surrounding image
+     */
+    imageWrapperStyle?: ViewStyle;
+
+    /**
+     * Add an image as the heading with the image prop
+     */
+    image?: ImageURISource;
+}
+
+/**
+ * Card component
+ *
+ * @see https://react-native-training.github.io/react-native-elements/API/card/
+ */
+export class Card extends React.Component<CardProps, any> {}
+
+/**
+ * Set the buttons within a Group.
+ */
+export interface ElementObject {
+    element: JSX.Element | React.ReactType;
+}
+
+/**
+ * Set the border styles for a component.
+ *
+ * @export
+ * @interface InnerBorderStyleProperty
+ */
+export interface InnerBorderStyleProperty {
+    color?: string;
+    width?: number;
+}
+
+export interface ButtonGroupProps {
+    /**
+     * Current selected index of array of buttons
+     */
+    selectedIndex: number;
+
+    /**
+     * Method to update Button Group Index
+     */
+    // onPress(): void;
+    onPress(selectedIndex: number): void;
+
+    /**
+     * Array of buttons for component, if returning a component, must be an object with { element: componentName }
+     */
+    buttons: string[] | ElementObject[];
+
+    /**
+     * Choose other button component such as TouchableOpacity
+     *
+     * @default TouchableHighlight
+     */
+    component?: React.ComponentClass;
+
+    /**
+     * Specify styling for main button container
+     */
+    containerStyle?: ViewStyle;
+
+    /**
+     * inherited styling	object (style)	specify styling for button
+     */
+    buttonStyle?: ViewStyle;
```

Comment:
All the instances of `ViewStyle, TextStyle, ImageStyle` need to be wrapped with the `StyleProp` from 'react-native`

e.g `StyleProp<ViewStyle>`. This is so someone can pass in an array of styles